easybuild/easyconfigs/c/Cellpose/Cellpose-2.2-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,56 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'Cellpose'
+version = '2.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = "https://www.cellpose.org/"
+description = """a generalist algorithm for cellular segmentation"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', True),
```

Comment:
```suggestion
    ('CUDA', '11.7.0', '', SYSTEM),
```

Reply:
Yeah. I'd not bothered fixing this as there is a wider PyTorch `1.12.0` vs `1.12.1` issue to decide on. See also #17183, #17217, and the easyconfigs for `cell2location` and `scvi-tools` that I've not yet PR-ed.